src/transaction_utils.cpp

```diff
@@ -473,6 +485,14 @@ namespace sdk {
                     if (net_params.liquid()) {
                         output["public_key"] = blinding_key_from_addr(address);
                     }
+                    const bool is_change = json_get_value(addressee, "is_change", false);
+                    if (is_change) {
+                        // This is an explicit change output (as opposed to one generated by the gdk)
+                        // Mark with 'is_change'
+                        // Can be distinguished from a generated change output because the change_index
+                        // will be set to -1
```

Comment:
As above this needs to be enforced by an assert

Reply:
No longer relevant